### PR TITLE
Incorrect image path for Hotspot background image WBL-24

### PR DESCRIPTION
**Pull request by rakeshqdsa**
https://learnosity.atlassian.net/browse/WBL-5

### CODE REVIEW

File: src/Processors/QtiV2/In/Constants.php [hunk 1]
```diff
@@ -25,4 +25,6 @@ class Constants
     public static $unsupportedInteractions = [
         'drawingInteraction'
     ];
+    
+    public static $learnosityImagePath = 'https://assets.learnosity.com/organisations/';
```
Comment (ttton) [inline src/Processors/QtiV2/In/Constants.php]: Can we rename this (for clarity and consistency) to `$baseLearnosityAssetsUrl`?

File: src/Processors/QtiV2/In/ItemMapper.php [hunk 2]
```diff
@@ -150,7 +152,11 @@ protected function buildLearnosityItemFromQtiAssessmentItem(AssessmentItem $asse
         if (isset($metadata['point_value'])) {
             $itemBuilder->setItemPointValue($metadata['point_value']);
         }
-
+        
+        if(isset($metadata['organisation_id'])){
```
Comment (ttton) [inline src/Processors/QtiV2/In/ItemMapper.php]: PSR-2:
* single space before opening parenthesis
* single space before opening brace

File: src/Processors/QtiV2/In/Constants.php (hunk 1 above)
Comment (rakeshqdsa) [reply src/Processors/QtiV2/In/Constants.php]: Done

File: src/Processors/QtiV2/In/ItemMapper.php (hunk 2 above)
Comment (rakeshqdsa) [reply src/Processors/QtiV2/In/ItemMapper.php]: Done

**Pull request merged**